### DISCUSSION TITLE
Update Debian repo and packages for Debian 11 Bullseye

### DISCUSSION THREAD

**Discussion by Knezzen** [SpectrumIM/spectrum2#408] [Q&A] [answered]
Hi!

With the new stable release of Debian there is a need to update the packages in the repository.
When in time will this happen? Would love to drop Arch in favor of Debian (again), hence my curiosity.

**Comment by vitalyster** [comment, marked as answer]
Bullseye packages are landed to our repository. You can also use our container image on any OS.

**Comment by coldtobi** [comment]
FWIIW, I'm currently working on packaging spectrum2 for Debian  (as in to be included in Debian)
